modules/enterprise/server/jar/src/main/java/org/rhq/enterprise/server/content/ContentManagerRemote.java

```diff
@@ -284,11 +283,13 @@ PackageTypeAndVersionFormatComposite findPackageTypeWithVersionFormat(Subject su
      * @param architectureId
      * @param temporaryContentHandle
      * @return the package version
-     *
      * @see ContentManagerRemote#createPackageVersionWithDisplayVersion(org.rhq.core.domain.auth.Subject, String, int, String, String, Integer, byte[])
-     *
      * @since 4.10
      */
     PackageVersion createPackageVersionWithDisplayVersion(Subject subject, String packageName, int packageTypeId,
-        String version, String displayVersion, Integer architectureId, String temporaryContentHandle);
+                                                          String version, String displayVersion, Integer architectureId,
+                                                          String temporaryContentHandle);
+
+    //To force to discovery certain package
+    void mergePackage(int resourceId, ResourcePackageDetails details);
```

Comment:
Because it's used by `ResourceClientProxy` and it needs this exposed as a remote method